assets/migrations/scripts/20190829154642_create_organization_table.sql

```diff
@@ -0,0 +1,39 @@
+--
+--    Copyright 2010-2016 the original author or authors.
+--
+--    Licensed under the Apache License, Version 2.0 (the "License");
+--    you may not use this file except in compliance with the License.
+--    You may obtain a copy of the License at
+--
+--       http://www.apache.org/licenses/LICENSE-2.0
+--
+--    Unless required by applicable law or agreed to in writing, software
+--    distributed under the License is distributed on an "AS IS" BASIS,
+--    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+--    See the License for the specific language governing permissions and
+--    limitations under the License.
+--
+
+-- // create_organization_table
+-- Migration SQL that makes the change goes here.
+
+CREATE TABLE team.organization
+(
+	id bigserial NOT NULL,	
+    identifier character varying NOT NULL,
+    active boolean default false,
+    name character varying NOT NULL,
+    type jsonb ,
```

Comment:
Curious, what is being stored in this field?  Also, could we remove the space before the comma?

Reply:
This is the organization terminology code system. I suspect we may not need in openSRP or Reveal however its important when exchanging data via FHIR.

The data to be stored is in the below format 
```
    "coding": [
      {
        "system": "http://terminology.hl7.org/CodeSystem/organization-type",
        "code": "team",
        "display": "Team"
      }
    ]
```
This is per the spec https://smartregister.atlassian.net/wiki/spaces/REV/pages/1159921677/Team+Structure+and+User+Assignment